### PR TITLE
Add a 'View decrypted source' button

### PR DESCRIPTION
**Pull request by richvdh**

### CODE REVIEW

**Comment by richvdh** [comment]
retest this please

**Comment by richvdh** [comment]
@dbkr: I think the builds are fixed now, even if jenkins/travis are being difficult about it.

File: src/components/views/context_menus/MessageContextMenu.js [hunk 1]
```diff
@@ -47,7 +47,16 @@ module.exports = React.createClass({
...
         var ViewSource = sdk.getComponent('structures.ViewSource');
         Modal.createDialog(ViewSource, {
-            mxEvent: this.props.mxEvent
+            content: this.props.mxEvent.event,
+        }, 'mx_Dialog_viewsource');
+        if (this.props.onFinished) this.props.onFinished();
+    },
+
+    onViewClearSourceClick: function() {
+        var ViewSource = sdk.getComponent('structures.ViewSource');
```
Comment (dbkr) [inline src/components/views/context_menus/MessageContextMenu.js]: Could probably be a const here

Comment (richvdh) [inline src/components/views/context_menus/MessageContextMenu.js]: done

**Pull request merged**